src/reducers/index.js

```diff
@@ -126,12 +126,18 @@ function reducers(state = getInitialState(), action) {
       const { newProps, oldProps } = action;
       let newState = setStateFromProps(state, newProps);
 
+      // recalculate row offsets if count or default row heights changed
       if (oldProps.rowsCount !== newProps.rowsCount ||
           oldProps.rowHeight !== newProps.rowHeight ||
           oldProps.subRowHeight !== newProps.subRowHeight) {
         newState = initializeRowHeightsAndOffsets(newState);
       }
 
+      // recalculate column offsets if count or default column widths changed
```

Comment:
fixes a bug where increase in number of column doesn't work (this is similar to how we reset the prefix tree for row offsets just below)